lib/pom.xml

```diff
@@ -13,103 +14,113 @@
 
   <build>
     <plugins>
-        <plugin>
-            <artifactId>maven-compiler-plugin</artifactId>
-            <configuration>
-                <source>1.8</source>
-                <target>1.8</target>
-            </configuration>
-        </plugin>
-        <plugin>
-            <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-dependency-plugin</artifactId>
-            <executions>
-                <execution>
-                    <goals>
-                        <goal>properties</goal>
-                    </goals>
-                </execution>
-            </executions>
-        </plugin>
-        <plugin>
-            <groupId>org.codehaus.mojo</groupId>
-            <artifactId>exec-maven-plugin</artifactId>
-            <version>1.6.0</version>
-            <executions>
-                <execution>
-                    <phase>generate-sources</phase>
-                    <goals>
-                        <goal>exec</goal>
-                    </goals>
-                    <configuration>
-                        <sourceRoot>${project.build.directory}/generated-sources/dgm</sourceRoot>
-                        <executable>java</executable>
-                        <arguments>
-                            <argument>-jar</argument>
-                            <argument>${com.cloudbees:groovy-cps-dgm-builder:jar:jar-with-dependencies}</argument>
-                            <argument>${project.build.directory}/generated-sources/dgm</argument>
-                        </arguments>
-                    </configuration>
-                </execution>
-            </executions>
-        </plugin>
+      <!-- TODO removing this results in NotBlockTest.serialFormBackwardsCompatibility:38 » InvalidClass com.cloudbees.groovy.cps.Continuation$Halt; local class incompatible: stream classdesc serialVersionUID = 1996175119530893094, local class serialVersionUID = 8436532706603657532 -->
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-site-plugin</artifactId>
-        <dependencies>
-          <dependency>
-            <groupId>org.kohsuke</groupId>
-            <artifactId>doxia-module-markdown</artifactId>
-            <version>1.0</version>
-          </dependency>
-        </dependencies>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
+          <testSource>1.8</testSource>
+          <testTarget>1.8</testTarget>
+          <release combine.self="override" />
+          <testRelease combine.self="override" />
+        </configuration>
       </plugin>
```

Comment:
Setting `release=11` failed a test with the error given in the comment, so overriding the values from the parent POM to continue to compile with `source=1.8` and `target=1.8` for now.